httpclient_android/http_client/src/main/java/com/backblaze/b2/client/webApiHttpClient/android/UrlConnection/B2StorageHttpClientBuilder.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2020, Backblaze Inc. All Rights Reserved.
+ * License https://www.backblaze.com/using_b2_code.html
+ */
+package com.backblaze.b2.client.webApiHttpClient.android.UrlConnection;
```

Comment:
the webApiHttpClient name refers to a webApi implementation using the Apache HttpClient library.

I think the correct package name here would be:
com.backblaze.b2.client.webApiUrlConnection

I would also advocate for making this a regular java module as there is nothing Android specific about this. If needed, create a second Gradle file to allow it to be built for Android.